src/CONST.js

```diff
@@ -1,5 +1,8 @@
+import lodashGet from 'lodash/get';
+import Config from 'react-native-config';
```

Comment:
Can't import the Config directly, Its `undefined` may be due to circular dependency.

Reply:
I tried resolving it via creating another APP_URL file but still that was returning `undefined`.